aws/ami/windows/windows.pkr.hcl

```diff
@@ -66,4 +67,25 @@ build {
     ]
   }
 
+  # Install CUDA Toolkits
```

Comment:
There is probably a way to make this so we don't have to copy/paste this so many times but my attempts to use the `dynamic` block here with a `for_each` loop proved not to work as intended